apps/web/src/components/Notifications/Notification.tsx

```diff
@@ -621,4 +670,12 @@ const Container = styled.div<{ isClickable: boolean }>`
           }
         `
       : css``};
+  ${({ hasMultipleTokens }) =>
+    hasMultipleTokens
+      ? `
+      :hover ${StyledPostPreview} {
+        border: 1px solid ${colors.faint};
+      }
+    `
+      : ''}
```

Comment:
the purpose of this is to ensure the border color changes for the multi-token children imgs when a user hovers over the parent notification container